enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -248,16 +246,37 @@ class AdjointGenerator
         Value *orig_op1 = FPMO->getOperand(0);
         bool constantval1 = gutils->isConstantValue(orig_op1);
```

Comment:
I'd change this to a return here if constant rather than having the repeated checks.

Reply:
done